.github/workflows/main.yml

```diff
@@ -0,0 +1,161 @@
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+  release:
+    types:
+      - created
+
+jobs:
+  E2E:
+    runs-on: ubuntu-latest
+    name: Install setup build e2e
+    steps:
+      - run: |
+          mkdir "$HOME/bin" && export PATH="$HOME/bin:$PATH"
+        shell: bash
+      - name: Checkout code
+        uses: actions/checkout@v2
+      - name: Set up Go
+        uses: actions/setup-go@v2
+        with:
+          go-version: 1.15
+      - name: Install Kubectl
+        uses: azure/setup-kubectl@v1
```

Comment:
azure? 🤔

Reply:
https://github.com/marketplace/actions/kubectl-tool-installer
It's a verified creator by Github and seemed legit. The alternative is curling to install kubectl.

Reply:
Changed it to curl